stock_customer_deposits/models/stock_picking.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        pickings = super().create(vals_list)
+        for picking in pickings:
+            if (
+                picking.picking_type_id.code == "incoming"
+                and picking.location_id.usage == "customer"
+            ):
+                picking.write({"owner_id": picking.partner_id.id})
```

Comment:
@EmilioPascual Mmmmh, is this causing the fact you put this module in rebels one ? This should be configurable. IMHO, an option on picking type should be the good way.

Reply:
It's not, I put this module in rebels because stock_owner_restriction is incompatible, so if i put stock_owner_restriction in excludes, the ci don't run.
However, it is a good idea that it is possible to configure in operation type. Thank you @rousseldenis

Reply:
> However, it is a good idea that it is possible to configure in operation type. Thank you @rousseldenis

I would say it should be mandatory as this changes standard flow for all IN pickings

Reply:
I changed that. Again than you @rousseldenis for your reviews and time.